### DISCUSSION TITLE
[Unstable] control command is butchered on npcs in mp

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#15078] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

controlling any friendly/hostile npc with the control console command makes it so the npc still has actual control but you're locked into its screen, it looks like you can try moving it but the actual position is still locked to the server side one (that's my guess)
https://cdn.discordapp.com/attachments/1244071644964851853/1303562966096150529/Control_command.mp4?ex=672cddde&is=672b8c5e&hm=542ef57645cfcee8cd4621a8d9e115c31141876a66b241fe915de2dd59cafb47&

### Reproduction steps

spawn any npc
control npc
try moving around

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in a future build!
